core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.spec.ts

```diff
@@ -238,4 +253,37 @@ describe('InputResponsePairComponent', () => {
       ExplorationPlayerConstants.AUDIO_HIGHLIGHT_CSS_CLASS
     );
   });
+
+  it('should return empty css class for feedback audio highlight ' +
+    'when audio player service is not playing', () => {
+    spyOn(audioTranslationManagerService, 'getCurrentComponentName')
+      .and.returnValue(AppConstants.COMPONENT_NAME_FEEDBACK);
+    spyOn(audioPlayerService, 'isPlaying').and.returnValue(false);
+    component.isLastPair = true;
+
+    expect(component.getFeedbackAudioHighlightClass()).toBe(
+      '');
```

Comment:
Why not?

```suggestion
    expect(component.getFeedbackAudioHighlightClass()).toBe('');
```

Sorry if it wasn't apparent before. Ditto elsewhere.

Reply:
I have done it. One recommendation though, while I do this, I am also wondering if you can point me to the relevant link where this is mentioned so that I/anyone-new can easily refer them (so that from future I can refer to them) because I made it according to how it exists currently in the codebase, and I don't think these things compromises too much on readability either. Is it possible we can make it a part of the Linter or Formatter (Visual Studio Code Formatter)? At best, this causes more confusion since there are two different styles followed in the code-base.

Reply:
We break lines when the statement exceeds the line length limit (which is 80 as defined [here](https://github.com/oppia/oppia/blob/905ad39a27996d9cb1bf40c334c0cddbc3b37f5a/.eslintrc#L260-L267)).